backend/app/views/spree/admin/users/edit.html.erb

```diff
@@ -23,9 +23,22 @@
         </div>
       <% end %>
     <% end %>
+
   </div>
 </fieldset>
 
+<fieldset id='forgot-password'>
```

Comment:
We should hide this conditionally 

```rb
if @user != try_spree_current_user
```

That this is only visible for admins that don't manage their own password.

Reply:
Why? That breaks consistency with the UI.